spec/takeaway_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'takeaway'
+
+RSpec.describe Takeaway do
+  let(:messager) {double :messager}
+  let(:menu) {double :menu, show_menu: {"chicken" =>3,
+     "rice" =>1,
+     "pizza" =>6,
+     "chips" =>2 }
+   }
```

Comment:
Lint/BlockAlignment: `}` at 9, 3 is not aligned with `let(:menu) {double :menu, show_menu: {"chicken" =>3,` at 5, 2.